packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -400,7 +400,9 @@ export default class QuanticCategoryFacet extends LightningElement {
   }
 
   get canShowLess() {
-    return this.facet && this.state?.canShowLessValues;
+    return (
+      this.facet && this.state?.canShowLessValues && !this.isFacetSearchActive
```

Comment:
We can't show less when the facetSearch is active?

Reply:
yes, I'm fixing the following bug, it didn't make sense to display the show less button when facet search is active:

Before: 

https://github.com/user-attachments/assets/7058f392-03ad-4894-8938-03620a71c82d

After: 


https://github.com/user-attachments/assets/f428905a-7f62-4d82-b3cb-63ec258ca144

It's the same behaviour we have for the show more values.
the show more button and the show less button control facet values, however when a facet search is active facet search values are displayed instead of facet values like can see in the video that's why I'm no longer displaying this button.